tsconfig.json

```diff
@@ -0,0 +1,43 @@
+{
+  "compilerOptions": {
+    "target": "ESNext",
+    "module": "ESNext",
+    "allowJs": true,
+    "checkJs": false,
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true,
+    "outDir": "./dist",
+    "rootDir": "./",
+    "strict": true,
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": true,
+    "strictBindCallApply": true,
+    "strictPropertyInitialization": true,
+    "noImplicitThis": true,
+    "alwaysStrict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "noUncheckedIndexedAccess": true,
```

Comment:
It might be nice to add `"noPropertyAccessFromIndexSignature": false, // used by trigger output strings` just as a comment about why that isn't turned on.  Assuming this is one of those json files that allows comments.  <_<

Reply:
It seems that this option is added in ts 4.2.0 and current latest stable version is 4.1.5

Reply:
Ah ok!  Thanks.  Sorry, I was just reading through options from the typescript page.